src/Extractor/HttpClient.php

```diff
@@ -115,39 +139,34 @@ public function fetch($url, $skipTypeVerification = false, $httpHeader = [])
         $this->logger->log('debug', 'Trying using method "{method}" on url "{url}"', ['method' => $method, 'url' => $url]);
 
         try {
+            /** @var ResponseInterface $response */
             $response = $this->client->$method(
                 $url,
                 [
-                    'headers' => [
-                        'User-Agent' => $this->getUserAgent($url, $httpHeader),
-                        // add referer for picky sites
-                        'Referer' => $this->getReferer($url, $httpHeader),
-                    ],
-                    'timeout' => $this->config['timeout'],
```

Comment:
Unfortunately, not easily. Each adapter has a different configuration. We must configure a client manually and inject it